src/index.ts

```diff
@@ -48,8 +48,9 @@ module.exports.ContentRepo = (
 	require("matrix-js-sdk").ContentRepo
 );
 
-module.exports.PrometheusMetrics = require("./components/prometheusmetrics");
-module.exports.PrometheusMetrics.AgeCounters = require("./components/agecounters");
+
+export * from "./components/prometheusmetrics";
+module.exports.PrometheusMetrics.AgeCounters = require("./components/agecounters").AgeCounters;
```

Comment:
That's ok without defining `module.exports.PrometheusMetrics` explicitly??
ok, i guess..

Reply:
Ah, it's the above export line. It's sugar syntax for `module.exports.PrometheusMetrics = require("./components/prometheusmetrics");`

Reply:
Just feels super weird to patch an object that has not been defined explicitly and hope the export name in the other file does not change.